Dockerfile.dev

```diff
@@ -16,10 +16,10 @@ RUN gem install bundler
 WORKDIR /usr/src/app
 
 COPY Gemfile Gemfile.lock ./
-RUN bundle install --full-index
+RUN bundle install
```

Comment:
I've taken this out and instead now check for updates to the `exercism_config` gem at boot-time.